hunts/src/HuntViewMain.js

```diff
@@ -195,6 +202,19 @@ export const HuntViewMain = (props) => {
             Unsolved
           </label>
         </div>
+        <div>
+          {filterTags.length > 0 ? 'Viewing puzzles with tags: ' : ''}
+          {filterTags.map(({ name, color, id }) => {
+            return <TagPill
+              name={name}
+              color={color}
+              id={id}
+              puzzleId={null}
+              key={name}
+              onDelete={function () {addOrRemoveFilterTag({ name, color, id })}}
```

Comment:
Do we intend to use only the "remove" part of `addOrRemoveFilterTag` here?

Reply:
Yeah, we do. I could create a separate function to do that if desired.

Reply:
Yeah, it might be clearer to separate the "find matching tag" and "remove" (might just be an array pop or something) logic. Then for `addOrRemove` you can use `findMatchingTag`, I think?

Reply:
Not quite sure I see how to do this elegantly but I can try (might not be doing what you thought of though).

Reply:
Eh, if it ends up being more complicated, feel free to leave as is and add a comment here mentioning that we expect to remove only :+1:

Reply:
I recently pushed a new version of the whole tagFilter.js file; the new version is longer but I think it's a lot clearer so I personally prefer it.